stable_baselines/common/vec_env/vec_normalize.py

```diff
@@ -88,48 +89,69 @@ def step_wait(self):
         where 'news' is a boolean vector indicating whether each element is new.
         """
         obs, rews, news, infos = self.venv.step_wait()
-        self.ret = self.ret * self.gamma + rews
         self.old_obs = obs
-        obs = self._normalize_observation(obs)
-        if self.norm_reward:
-            if self.training:
-                self.ret_rms.update(self.ret)
-            rews = np.clip(rews / np.sqrt(self.ret_rms.var + self.epsilon), -self.clip_reward, self.clip_reward)
+        self.old_rews = rews
+
+        if self.training:
```

Comment:
@araffin do you think we should update observation statistics even if `self.norm_obs` is false? It might have a slight resource overhead but otherwise can't see a disadvantage.

Reply:
Decided against this. Don't have strong feelings either way but the resource overhead seems minimal and I could imagine use cases where this is actually helpful (e.g. running two policies, one of which wants normalized and other not, and still want to update observation statistics from both).